app/jinja_filters.py

```diff
@@ -383,21 +363,21 @@ def map_relationships_config_processor():
     return dict(map_relationships_config=map_relationships_config)
 
 
-class SelectOptionConfig:
+class DropdownConfig:
     def __init__(self, option, select):
-        self.value, self.text = option
+        self.value, self.text = option.value, option.label
         self.selected = select.data == self.value
         self.disabled = self.value == "" and select.flags.required
 
 
 @blueprint.app_template_filter()
-def map_select_config(select):
-    return [SelectOptionConfig(choice, select) for choice in select.choices]
+def map_dropdown_config(select):
+    return [DropdownConfig(choice, select) for choice in select.choices]
```

Comment:
choices have `value=''` for `label= select an answer` , that is not one of the options/choices to select from  , why is it under choices ?

Reply:
If I understood your question correctly, then that is cause dropdown have a default value to chose from which is `disabled`, it's more of a placeholder.